### ISSUE TITLE
Add Faction Alignment

### ISSUE THREAD

**Issue by JonathanArendt** [haggen/localthreat#64]
**What's your idea?**
Add an optional row for which Faction the pliot is in, IE, Guristas, Caldari, Angel Cartel, ect.
...

**Why do you think it's a good idea?**
To assist in identifying which pilots are war targes based on you're faction.
...

**Are there alternatives that achieve the same goal?**
Manualy checking in game, or checking the players individually.
...

**Comment by haggen** [comment]
@JonathanArendt I just released a big update. If you could confirm that the Faction column was what you wanted I'd appreciate it. Thanks.

**Comment by haggen** [comment]
@e1vis25 Is the update satisfactory to your request?

**Comment by e1vis25** [comment]
Perfect. Great update. Thanks!

**Comment by JonathanArendt** [comment]
I'm sorry for not responding , I haven't been on EVE for a while, and I don't have a way to see the changes, can you include a picture of what it looks like, or share a sample copy paste?

V/R

Jonathan Arendt
________________________________

**Comment by haggen** [comment]
@JonathanArendt Certainly.

If you wanna see it yourself: https://localthreat.xyz/6lW9mWVeSKUU

<img width="1334" height="886" alt="Image" src="https://github.com/user-attachments/assets/ed63b1fe-1ae2-4c4d-a4f5-20cf3719068b" />

**Comment by JonathanArendt** [comment]
YES!!! That is exactly what I was asking for! It helps in faction warfare to quickly see who is under which alliance! Now does it also show pirate factions? I'm assuming it does just like the Empire Factions just wanted to clarify.

V/R

Jonathan Arendt
________________________________

**Comment by haggen** [comment]
@JonathanArendt Yes it does show pirate factions as well. Thanks for the feedback, I'll close the issue now. Cheers!
